### PR TITLE
changes to DriveStrength

### PR DESCRIPTION
**Pull request by R3dst0ne**

### CODE REVIEW

File: src/main/java/com/cburch/logisim/fpga/data/DriveStrength.java [hunk 1]
```diff
@@ -9,12 +9,11 @@
 
 package com.cburch.logisim.fpga.data;
 
-import java.util.Iterator;
 import java.util.LinkedList;
 import java.util.List;
 
 public class DriveStrength {
-  public static String GetContraintedDriveStrength(char id) {
+  public static String getContraintedDriveStrength(char id) {
```
Comment (davidhutchens) [inline src/main/java/com/cburch/logisim/fpga/data/DriveStrength.java:16]: I am not familiar with an English word "contrainted". Is that a technical term (I am not an expert on fpgas)? Or is there a better word (such as constrained)?

Comment (zdimension) [reply src/main/java/com/cburch/logisim/fpga/data/DriveStrength.java:16]: This is definitely a typo, it should be "constrained"

Comment (R3dst0ne) [reply src/main/java/com/cburch/logisim/fpga/data/DriveStrength.java:16]: I completely missed this one ^^'

**Review by maehne — APPROVED** [review]
Looks good to me. Thanks @R3dst0ne!

**Pull request merged**